include/arch/syspage-imx6ull.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * HAL syspage for i.MX 6ULL
+ *
+ * Copyright 2021 Phoenix Systems
+ * Authors: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _PHOENIX_SYSPAGE_IMX6ULL_H_
+#define _PHOENIX_SYSPAGE_IMX6ULL_H_
+
+
+typedef struct {
+
```

Comment:
We need a dummy field here, empty struct is not allowed by standard C (it's allowed as GCC extension). So maybe `int dummy;`?

Reply:
Done for `imx6ull` and `zynq7000`